src/app/pages/chat/chat-room-page/components/header/index.tsx

```diff
@@ -1,38 +1,40 @@
-import Icon from "@app/common/components/icons";
-import OptionsMenu from "@app/pages/chat/components/option-menu";
+// import Icon from "@app/common/components/icons";
+// import OptionsMenu from "@app/pages/chat/components/option-menu";
 import {
-  Action,
-  Actions,
-  actionStyles,
-  Avatar,
+  // Action,
```

Comment:
please remove the commented code. this is not relevant and making the file not readable. you can also use eslint setting.